plugins/gpio/GPIODriver.cpp

```diff
@@ -189,7 +201,15 @@ bool GPIODriver::UpdateGPIOPins(const DmxBuffer &dmx) {
 
     // Change the pin state if required.
     if (action != NO_CHANGE) {
-      char data = (action == TURN_ON ? '1' : '0');
+      char data;
```

Comment:
This feels a little repetitive to me. I wonder if it would be better to either calculate the boolean state, then optionally invert it if required, then convert the boolean into a char. Or I was going to suggest chaining ternary operators, but that doesn't feel great either to me.

Reply:
This looks much better thanks!